packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -116,23 +124,29 @@ module.exports = {
     // Set the property code.
     user.resetPasswordToken = resetPasswordToken;
 
-    // Send an email to the user.
-    const template = `
-      <p>We heard that you lost your password. Sorry about that!</p>
+    const settings = strapi.plugins['users-permissions'].config.email['reset_password'].options;
 
-      <p>But don’t worry! You can use the following link to reset your password:</p>
-
-      <p>${url}?code=${resetPasswordToken}</p>
+    const compiledMessage = _.template(settings.message);
```

Comment:
You should create a service to manage the templating of each mail. The service could template and compile the template with the variables defined in parameters. It will be easier to maintain.

Also, the variables should be uppercased.